stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/EmojiUtils.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.getstream.sdk.chat.utils
+
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+@InternalStreamChatApi
+public object EmojiUtils {
+
+    public fun isEmojiOnly(message: String): Boolean {
```

Comment:
I'd add docs here for all the functions and the class just so that people understand what's going on (if they intend to use it)